src/components/base/BaseCheckbox.vue

```diff
@@ -0,0 +1,89 @@
+<script setup lang="ts">
+import { defineEmits } from "vue";
+const emit = defineEmits(["update:modelValue", "blur"]);
+const props = withDefaults(
+  defineProps<{
+    modelValue?: boolean;
+    errorMessage?: string;
+  }>(),
+  {
+    modelValue: false,
+    errorMessage: "",
+  }
+);
+
+const handleInput = () => {
+  emit("update:modelValue", !props.modelValue);
```

Comment:
Why you're emitting the opposite value here?

Reply:
As it is a checkbox if is boolean: either true - checked / false - unchecked. When it's clicked should reverse the state?